server/views/pages/explore.njk

```diff
@@ -75,5 +75,9 @@
     </div>
   </div>
 
+  <div data-component-name="Tweet block" data-component-id="explore">
```

Comment:
As in, 'The "Tweet block" that appears on the explore page' – wasn't sure if there's a better way of differentiating between them. I'm open to suggestions.